glean-core/csharp/csharp/Glean.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Authors>Mozilla</Authors>
+    <RepositoryUrl>https://github.com/mozilla/glean</RepositoryUrl>
+    <Description>The Glean SDK is a modern approach for a telemetry library by Mozilla.</Description>
+    <Version>0.0.1</Version>
+    <RootNamespace>Mozilla.Glean</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Make sure to pack the glean-core DLL in the final package -->
+    <!-- TODO: pick the appropriate DLL depending on the target -->
+    <Content Include="../../../target/debug/glean_ffi.dll">
```

Comment:
My `glean_ffi.dll` can't be added via glean/Dependencies/Add Reference. I was thinking they only allow to import the dll is written by C#.

Reply:
What do you mean?

Reply:
I guess <Pack> and <PackagePath> are hand-writing by yourself because I can't find a way to add them in MSVC. I think adding them looks right to me because we will pack them into NuGet package for sure.

Besides that, using `$(ProjectDir)` predefined macros of VS here would be more standard when including external files.

Reply:
Yes, this will be solved in an upcoming PR.

Reply:
I see. I will take care of this in [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1643564).